lorien/UI/Dialogs/KeyBindingsList.gd

```diff
@@ -13,20 +13,29 @@ func _ready() -> void:
 	_add_key_dialog.connect("hide", self, "_bind_key_dialog_hidden")
 	GlobalSignals.connect("language_changed", self, "_populate_input_list")
 
+# -------------------------------------------------------------------------------------------------
+func _second_element(a: Array, b: Array) -> bool:
```

Comment:
Could you change that to `_compare_second_element`?

Reply:
Sorry, a bit slow to respond right now. Lot's of stuff on my place ;-)

Reply:
Don't worry, no problem. Thanks!